backend/faq/views.py

```diff
@@ -97,13 +100,49 @@ def get(self, request):
         try:
             high_level_instance = HighLevelModel.objects.get(user_id=request.user.id)
             faq_data = list(FAQ.objects.filter(high_level_id=high_level_instance.id).values("question", "answer"))
-            download_csv_file(faq_data)
+            message = download_csv_file(faq_data)
             return Response(
-                {
+                {    
+                    "file": message,
                     "details": "download csv file",
                     "success": True
                 },
                 status=status.HTTP_200_OK)
-          
         except Exception:
-            return Response("Please integrate your high level Integrations")
\ No newline at end of file
+            return Response("Please integrate your high level Integrations")
+
+
+class ImportFAQFile(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        try:
+            high_level_instance = HighLevelModel.objects.get(user_id=request.user.id)
+            filename = request.data.get('file')
+            file, extension = os.path.splitext(str(filename))
+            if extension != '.csv':
+                return Response(
+                 {
+                    "message": "Please select a .csv file only",
+                    "success": False
+                 },
+                 status=status.HTTP_400_BAD_REQUEST)
+            delete_exiting_faq = request.data.get('delete_exiting_faq', None)
+            if delete_exiting_faq == "yes":
+                FAQ.objects.filter(high_level_id=high_level_instance.id).delete()
+                
+
+            df = pd.read_csv(filename)
+            data = df.to_numpy()
+            for x in data:
```

Comment:
It can be done in a much faster way by using pandas, By iterrows or to_dict method